haskell-cabal.el

```diff
@@ -488,7 +489,10 @@ OTHER-WINDOW use `find-file-other-window'."
                           (string= val "{")
                           (not val))
                       (push projectName matches)
-                    (push val matches))))
+                    (push (if (eq 'stack-ghci process-type)
+                              (concat projectName ":" val)
```

Comment:
I can switch it to `package-name` which might be more appropriate while I'm making edits tonight.